### ISSUE TITLE
the difference and meaning

### ISSUE THREAD

**Issue by Light--** [jacobgil/pytorch-grad-cam#90]
hi, @jacobgil , thanks for sharing!

Now i want to use gradcam method to visualize the activation map of my model. But after reading your code, it will produce 3 image results. May i ask what's the difference and meaning of them(cam_image, gb, cam_gb), respectively?
1. f'{args.method}_cam.jpg', cam_image
2. f'{args.method}_gb.jpg', gb
3. f'{args.method}_cam_gb.jpg', cam_gb

also, may i ask what's the function of `gb_model`?  just use it to see the gradients? then what's the difference of gb and cam_gb?
`gb_model = GuidedBackpropReLUModel(model=model, use_cuda=args.use_cuda)`

**Comment by jacobgil** [comment]
Hi,

1. cam_image: Is the class activation map heatmap, visualized on top of the image.
2. gb: Guided backpropagation (https://www.cs.toronto.edu/~guerzhoy/321/lec/W07/HowConvNetsSee.pdf) 
This is just backpropagating through the target back to the input image, and keeping only positive gradients.
3. It's the cam heat map, multiplied by the gb.
This was suggested in the original grad-cam paper.
This keeps the nice gradient visualization from gb, but keeps only the class-discriminative part, e.g only the cat but not the dog.

**Comment by Light--** [comment]
May i ask how to use your code to visualize Mxnet models' attention? @jacobgil
